src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -760,7 +760,8 @@ private function addService(string $id, Definition $definition): array
                 $return[] = '';
             }
 
-            $return[] = sprintf('@deprecated %s', $definition->getDeprecationMessage($id));
+            $deprecation = $definition->getDeprecation($id);
+            $return[] = sprintf('@deprecated %s%s', $deprecation['version'] ? ('Since ' . ($deprecation['package'] ? $deprecation['package'] . ' ' : '') . $deprecation['version'] . ': ') : '', $deprecation['message']);
```

Comment:
we should borrow the logic from https://github.com/symfony/deprecation-contracts/blob/master/function.php#L25

Reply:
Done